script_runner/tasks.py

```diff
@@ -216,10 +222,13 @@ def execute(script_path, ctx, process):
                                bufsize=1,
                                close_fds=on_posix)
 
-    return_code = None
+    consumers = {
+        'stdout': OutputConsumer(ctx, process.stdout, 'stdout'),
+        'stderr': OutputConsumer(ctx, process.stderr, 'stderr')
+    }
 
-    stdout_consumer = OutputConsumer(process.stdout)
-    stderr_consumer = OutputConsumer(process.stderr)
+    ctx.logger.info(', '.join(['{0} -> {1}'.format(key, value.out_file)
+                               for key, value in consumers.iteritems()]))
```

Comment:
drop the `[]` in `.join`, making it simply `', '.join('{0}' for ...)` (a generator can be used there, doesn't need to be a list)

Reply:
since `consumers` is always just 2 elements, let's use `.items()` instead of `.iteritems()` as to not introduce additional py3-incompatible lines

Reply:
@tehasdf will be fixed shortly.